webui/react/src/omnibar/Omnibar.tsx

```diff
@@ -19,12 +19,11 @@ const omnibarInput = () =>
   document.querySelector('#omnibar input[type="text"]') as HTMLInputElement | null;
 
 const Omnibar: React.FC = () => {
```

Comment:
Is there a reason the omnibar state can't just be local state in this component? Does the keydown listener have to be in `App`?

Reply:
definitely works fine when it's just local state here. 

my guess is that it was placed in `App` alongside `useKeyTracker`, but there's really no need to have both there, supported by Context. will update this PR to remove `OmnibarContext` and `OmnibarProvider` and use local state instead.